tomb-common/src/utils/tests/mod.rs

```diff
@@ -87,7 +87,7 @@ pub async fn setup_memory_test(
 }
 
 /// Create all of the relevant objects, using real BlockStores and real data
-pub async fn setup_test<TBS: TombBlockStore>(
+pub async fn setup_test<TBS: RootedBlockStore>(
     test_name: &str,
     metadata: TBS,
```

Comment:
nit:
acronyms for generics should ether not be acronyms (ex: `M`, `MetadataStore`) or descriptive acronyms (`RBS`, rather than `TBS`)